.devcontainer/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM docker.io/geosx/ubuntu20.04-gcc10:203-819
```

Comment:
Is there a need for `python3.x` or explicit version of `git` ? I guess it is already bundled unto the import. Would it be needed to add few ines of comment to remind versions that are such bundled and that we rely on

Reply:
I don't think we have any specific needs w.r.t. any version of git or python (we have something with brew, but it's another story.)